tuxemon/core/client.py

```diff
@@ -261,7 +269,10 @@ def update(self, time_delta):
         # Android-specific check for pause
         # if android and android.check_pause():
         #     android.wait_for_resume()
-
+        
```

Comment:
add this to the main loop here: https://github.com/Tuxemon/Tuxemon/blob/development/tuxemon/core/client.py#L241
follow the same format as `time_since_draw` and `frame_length` and make a timer for 60 seconds so that it is not making this call every frame.